internal/backend/db/dbgorm/sessions.go

```diff
@@ -67,6 +67,8 @@ func (db *gormdb) CreateSession(ctx context.Context, session sdktypes.Session) e
 
 	s := scheme.Session{
 		SessionID:        session.ID().String(),
+		BuildID:          session.BuildID().String(),
+		EnvID:            session.EnvID().String(),
```

Comment:
adding a comment here since I don't know how to add a comment on unchanged parts :)
what about listSessions and filters?
- maybe we should add to the filter buildID and envID?
- sessions without deployment will remain undeleted, and will be shown always in list. So maybe we should delete sessions via deleteBuild now?

Reply:
don't think we should delete sessions from delete builds. imo we shouldn't be deleting anything, we'll discuss.

envID is already in the filter. I'll add build as well.